test/weights.jl

```diff
@@ -36,6 +36,34 @@ weight_funcs = (weights, aweights, fweights, pweights)
     @test sum(sa, wv) === 7.0
 end
 
+# isequal and ==
+
+@testset "isequal" begin
+    x = FrequencyWeights([1; 2; 3], 6)
```

Comment:
It would be more natural to use commas, semicolons are only needed for concatenation of vectors.